front/ui/ui-charts/src/trackOccupancyDiagram/components/helpers/drawElements/drawOccupancyZones.ts

```diff
@@ -114,12 +107,13 @@ const drawOccupationZone = (
   ctx.font = '400 10px IBM Plex Mono';
 
   const { getTimePixel, getSpacePixel } = stcContext;
-  const yStart = getSpacePixel(position);
+  const yStart = getCrispLineCoordinate(getSpacePixel(position), BACKGROUND_HEIGHT);
```

Comment:
What is this yStart ?

Reply:
`yStart` is the top y coordinate of the flat step that contains the occupations zones layer.